docs/src/getting-started/cli.md

```diff
@@ -60,6 +60,7 @@ miden-faucet start \
 | `--remote-tx-prover-url` | Remote transaction prover | - | No |
 | `--api-keys` | Comma-separated API keys | - | No |
 | `--enable-otel` | Enable OpenTelemetry | `false` | No |
+| `--batch-size` | Maximum amount of requests to process in each batch. It sets how many p2id notes to create in each transaction | `8` | No |
```

Comment:
nit: I'd write this as "Maximum number of P2ID notes to create per transaction."